Airwallex/Card/Internal/AWXCardViewController.swift

```diff
@@ -0,0 +1,669 @@
+        sw.addTarget(self, action: #selector(addressSwitchChanged(_:)), for: .valueChanged)
+        return sw
+    }()
+
+    lazy var addressLabel: UILabel = {
+        let lb = UILabel()
+        lb.text = NSLocalizedString("Same as shipping address", comment: "Same as shipping address")
+        lb.textColor = UIColor.airwallexSecondaryText
+        lb.font = UIFont.airwallexSubhead1
+        return lb
+    }()
+
+    lazy var warningView: AWXWarningView = .init(
+        message:
+        NSLocalizedString("For UnionPay, only credit cards can be saved. Click “Pay” to proceed with a one time payment or use another card if you would like to save it for future use.", comment: "For UnionPay, only credit cards can be saved. Click “Pay” to proceed with a one time payment or use another card if you would like to save it for future use.")
+    )
+
+    lazy var closeButton: UIButton = {
+        let btn = UIButton()
+        btn.setImage(UIImage(named: "close", in: Bundle.resource()), for: .normal)
+        btn.addTarget(self, action: #selector(close), for: .touchUpInside)
+        return btn
+    }()
+
+    var currentBrand: AWXBrandType?
+    var paymentMethodType: AWXPaymentMethodType?
+
+    var saveCard: Bool = false
+
+    override public func viewDidLoad() {
+        super.viewDidLoad()
+        setupViews()
+    }
+
+    override public func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        Risk.log(event: "show_create_card", screen: "page_create_card")
+        registerKeyboard()
+    }
+
+    override public func viewWillDisappear(_ animated: Bool) {
+        super.viewWillDisappear(animated)
+        unregisterKeyboard()
+    }
+
+    func startIndicator() {
+        startAnimating()
+        confirmButton.isEnabled = false
+    }
+
+    func stopIndicator() {
+        stopAnimating()
+        confirmButton.isEnabled = true
+    }
+
+    func setupViews() {
+        navigationItem.leftBarButtonItem = UIBarButtonItem(
+            image: UIImage(named: "close", in: Bundle.resource()), style: .plain, target: self,
+            action: #selector(goBack)
+        )
+        closeButton.isHidden = navigationController != nil
+
+        enableTapToEndEditing()
+        view.backgroundColor = .white
+
+        view.addSubview(scrollView)
+        scrollView.addSubview(container)
+
+        NSLayoutConstraint.activate([
+            scrollView.leadingAnchor.constraint(equalTo: view.leadingAnchor),
+            scrollView.trailingAnchor.constraint(equalTo: view.trailingAnchor),
+            scrollView.topAnchor.constraint(equalTo: view.safeAreaLayoutGuide.topAnchor),
+            scrollView.bottomAnchor.constraint(equalTo: view.safeAreaLayoutGuide.bottomAnchor),
+
+            container.leadingAnchor.constraint(equalTo: scrollView.leadingAnchor),
+            container.trailingAnchor.constraint(equalTo: scrollView.trailingAnchor),
+            container.widthAnchor.constraint(equalTo: scrollView.widthAnchor),
```

Comment:
Is this needed since we've already leading and trailing?

Reply:
Yeah, otherwise the ScrollView cannot determine if it is horizontally scrollable.